mysql-test/suite/galera/t/mysql-wsrep#332.test

```diff
@@ -111,3 +111,29 @@ SELECT * FROM c;
 DROP TABLE c;
 DROP TABLE p1;
 DROP TABLE p2;
+
```

Comment:
Consider putting this test case in a separate file.
Having a separate file has several advantages:
- The test cases remain smaller
- Should this test fail, it might be easier to identify the failing part.
- A separate file gives it a specific name, for example MDEV-27276.test. So that if the test should fail, I won't lookup mysql-wsrep#322, instead I will lookup directly MDEV-27276